addok/shell.py

```diff
@@ -49,11 +49,15 @@ def _init_history_file(self):
             atexit.register(self.save_history)
 
     def save_history(self):
-        readline.write_history_file(self.history_file)
+        try:
+            readline.write_history_file(self.history_file)
+        except FileNotFoundError:
+            print(red('Unable to write history file to '
+                      '{}.'.format(self.history_file)))
 
     @property
     def history_file(self):
-        return str(Path(config.LOG_DIR).joinpath(self.HISTORY_FILE))
+        return str(Path.home() / self.HISTORY_FILE)
```

Comment:
Why [not](https://docs.python.org/3.5/library/pathlib.html#operators)? :)
BTW, we are already using it here and there: https://github.com/addok/addok/blob/master/addok/helpers/scripts.py#L9 for example :)